aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -165,30 +187,33 @@ export const ShippingExample = () => {
             validate="blur"
             value={formValues}
             onChange={setFormValues}
+            messages={{
+              required: '! This is required',
+            }}
             onSubmit={({ value, touched }) => onSubmit({ value, touched })}
           >
             <Box>
               <Heading level={4} margin={{ bottom: 'small', top: 'none' }}>
                 Shipping Information
               </Heading>
               <Text margin={{ horizontal: 'small', vertical: 'xsmall' }}>
-                Shipping Address
+                Shipping Address *
```

Comment:
This pattern seems a little suspicious from an accessibility perspective since we are not placing a label on the actual formfields. Not to be tackled in this PR, but would be a good grommet investigation about proper behavior for grouped formfields like this.

Reply:
Yes I also was not a fan of this I put it there to match the designs but should be more discussion around how we group things in a `formfield` and still meet accessibility